lab_1_classify_by_unigrams/main.py

```diff
@@ -65,11 +106,18 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     In case of corrupt input arguments, None is returned
     """
 
+    # checking input
```

Comment:
delete comments, in laboratory works they are excess

Reply:
all deleted now